sashimi-webapp/src/logic/filemanager/file.js

```diff
@@ -0,0 +1,133 @@
+import storage from '../../database/storage';
```

Comment:
You may consider changing this to `src/database/storage`. 

Our webpack is configured to resolve the `src` path.